Lib/_pydecimal.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2004 Python Software Foundation.
+# Copyright (c) 2017 Python Software Foundation.
```

Comment:
Maybe change this PR to strip the date range from the first one and leave the other alone? @VanL can you weigh in?